airbyte-webapp/src/components/base/Button/index.stories.tsx

```diff
@@ -1,25 +0,0 @@
-import { ComponentStory, ComponentMeta } from "@storybook/react";
-
-import Button from "./Button";
-
-export default {
-  title: "Ui/Button",
-  component: Button,
-  argTypes: {
-    backgroundColor: { control: "color" },
-  },
-} as ComponentMeta<typeof Button>;
-
-const Template: ComponentStory<typeof Button> = (args) => <Button {...args} />;
```

Comment:
I guess we definitely need story for storybook since it's a base component

Reply:
working on it